examples/tutorials/device_avsr_tutorial.py

```diff
@@ -0,0 +1,526 @@
+"""
+Device AV-ASR with Emformer RNN-T
+=================================
+
+**Author**: `Pingchuan Ma <pingchuanma@meta.com>`__, `Moto
+Hira <moto@meta.com>`__, `Zhaoheng Ni <zni@meta.com>`__, `Jeff
+Hwang <jeffhwang@meta.com>`__, `Xiaohui
+Zhang <xiaohuizhang@meta.com>`__, `Stavros
+Petridis <stavrosp@meta.com>`__, `Yumeng Tao <yumeng@meta.com>`__, `Maja
+Pantic <majapantic@meta.com>`__
+
+This tutorial shows how to run on-device audio-visual speech recognition
+(AV-ASR, or AVSR) with TorchAudio on a streaming device input,
+i.e. microphone on laptop. AV-ASR is the task of transcribing text from
+audio and visual streams, which has recently attracted a lot of research
+attention due to its robustness against noise.
+
+.. container:: alert alert-info
+
+   .. raw:: html
+
+      <h4>
+
+   Note
+
+   .. raw:: html
+
+      </h4>
+
+   .. raw:: html
+
+      <p>
+
+   This tutorial requires ffmpeg, sentencepiece, mediapipe,
+   opencv-python and scikit-image libraries. There are multiple ways to
+   install ffmpeg libraries. If you are using Anaconda Python
+   distribution, ``conda install -c conda-forge 'ffmpeg<7'`` will
+   install compatible FFmpeg libraries. You can run
+   ``pip install sentencepiece mediapipe opencv-python scikit-image`` to
+   install the other libraries mentioned.
+
+   .. raw:: html
+
+      </p>
+
+   .. raw:: html
+
+      <p>
+
+   .. raw:: html
+
+      </p>
+
+   We do not have any pre-trained models available at this time. The
+   following recipe use placedholder for the sentencepiece model path
+   ``spm_model_path`` and the pretrained model path ``avsr_model_path``.
+   If you are interested in the training recipe for real-time AV-ASR
+   models (AV-ASR), it can be found at `real-time
+   AV-ASR <https://github.com/pytorch/audio/tree/main/examples/avsr>`__
+   recipe.
+
```

Comment:
```suggestion
.. note::

   This tutorial requires ffmpeg, sentencepiece, mediapipe,
   opencv-python and scikit-image libraries.

   There are multiple ways to install ffmpeg libraries.
   If you are using Anaconda Python
   distribution, ``conda install -c conda-forge 'ffmpeg<7'`` will
   install compatible FFmpeg libraries.

   You can run
   ``pip install sentencepiece mediapipe opencv-python scikit-image`` to
   install the other libraries mentioned.

   We do not have any pre-trained models available at this time. The
   following recipe use placedholder for the sentencepiece model path
   ``spm_model_path`` and the pretrained model path ``avsr_model_path``.
   If you are interested in the training recipe for real-time AV-ASR
   models (AV-ASR), it can be found at `real-time
   AV-ASR <https://github.com/pytorch/audio/tree/main/examples/avsr>`__
   recipe.
```